client/app/src/main/java/com/microsoft/research/karya/ui/registration/SendOTPActivity.kt

```diff
@@ -49,6 +52,7 @@ class SendOTPActivity : NetworkActivity(
      * Start the next activity
      */
     override fun startNextActivity() {
-        startActivity(Intent(applicationContext, OTPActivity::class.java))
+        setResult(RESULT_OK)
+        onBackPressed()
```

Comment:
Don't use these overriden methods as they belong to `BaseActivity`.

Reply:
This is required in order to use NetworkActivity and make network calls. As we take out the functionality from NetworkActivity, this would be fixed